src/components/modals/RequestFormModal.vue

```diff
@@ -222,59 +226,61 @@ const RequestFormModal = defineComponent({
     }
   },
 })
-
-export default RequestFormModal
 </script>
 
 <style scoped lang="scss">
-.app-form__info {
-  &-wrapper {
+.app-form {
```

Comment:
if you rewrite global class, you should do it by use "bem component class"
for example in this camse you should wrote "request-form-modal__form"